spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/Mqttv5PahoMessageDrivenChannelAdapter.java

```diff
@@ -384,4 +463,13 @@ public void acknowledge() {
 
 	}
 
+	private class MessageListener implements IMqttMessageListener {
```

Comment:
Please, add an explicit `package protected` ctor to this class.
Otherwise compiler provides a lot of stuff for this kind of `private` ctor by default 😄